test/stub-test.js

```diff
@@ -940,6 +940,18 @@ describe("stub", function () {
             assert.contains(stub.firstCall.toString(), "not implemented");
         });
 
+        it("creates a non empty error message when error is a string and no message is passed", function () {
+            const stub = createStub()
+
+            stub.withArgs(1).throws("apple pie")
+
+            assert.exception(function () {
+                stub(1)
+            }, {
+                message: "Sinon-provided apple pie"
+            })
```

Comment:
Since the passed string is the actual error name, maybe use something that sounds like an error? Just in case someone casually looks at this for reference, it might confuse more than if you had passed something like "ValidationError".